src/main/kotlin/WorkerBuilder.kt

```diff
@@ -72,3 +79,17 @@ data class WorkerBuilder<T> private constructor(
     }
 }
 
+private const val THREAD_POOL_CORE_SIZE: Int = 1
+private const val THREAD_POOL_MAX_SIZE: Int = 2
+private const val THREAD_POOL_KEEP_ALIVE_TIME_SECONDS: Long = 30
+
+private fun createDefaultScope(): CoroutineScope {
+    val dispatcher: CoroutineDispatcher = ThreadPoolExecutor(
+        THREAD_POOL_CORE_SIZE,
```

Comment:
Im confused by this name. I know it is what kotlin calls it.
Comments to explain the different constants might be a useful addition.

Reply:
Or maybe I should just rename the constant so a comment is not needed? :thinking:

Reply:
Either works, as long as the purpose of the variable / constant is clear from just looking at it.